sdk/containerregistry/Azure.Containers.ContainerRegistry/tests/ContainerRepositoryClientLiveTests.cs

```diff
@@ -2,147 +2,183 @@
-                    CanDelete = true
-                });
-
-            RepositoryProperties properties = await _client.GetPropertiesAsync();
-
-            Assert.IsTrue(properties.WriteableProperties.CanList);
-            Assert.IsTrue(properties.WriteableProperties.CanRead);
-            Assert.IsTrue(properties.WriteableProperties.CanWrite);
-            Assert.IsTrue(properties.WriteableProperties.CanDelete);
-        }
-
-        public async Task InitializeTagProperties()
-        {
-            await _client.SetTagPropertiesAsync(
-                _tagName,
-                new ContentProperties()
-                {
-                    CanList = true,
-                    CanRead = true,
-                    CanWrite = true,
-                    CanDelete = true
-                });
-
-            RepositoryProperties properties = await _client.GetPropertiesAsync();
-
-            Assert.IsTrue(properties.WriteableProperties.CanList);
-            Assert.IsTrue(properties.WriteableProperties.CanRead);
-            Assert.IsTrue(properties.WriteableProperties.CanWrite);
-            Assert.IsTrue(properties.WriteableProperties.CanDelete);
+                    SourceImage = "library/hello-world",
+                    RegistryUri = "registry.hub.docker.com"
+                };
+
+                await _registryClient.Registries.ImportImageAsync(
+                    resourceGroupName: TestEnvironment.ResourceGroup,
+                    registryName: TestEnvironment.Registry,
+                    parameters:
+                        new ImportImageParameters
+                        {
+                            Mode = ImportMode.Force,
+                            Source = importSource,
+                            TargetTags = new List<string>()
+                            {
+                            $"library/hello-world:{tag}"
+                            }
+                        });
+            }
         }
 
         [RecordedTest]
         public async Task CanGetRepositoryProperties()
         {
-            RepositoryProperties properties = await _client.GetPropertiesAsync();
+            // Arrange
+            ContainerRepositoryClient client = CreateClient();
+
+            // Act
+            RepositoryProperties properties = await client.GetPropertiesAsync();
 
+            // Assert
             Assert.AreEqual(_repositoryName, properties.Name);
             Assert.AreEqual(new Uri(TestEnvironment.Endpoint).Host, properties.Registry);
         }
 
         [RecordedTest, NonParallelizable]
-        public async Task CanSetRepositoryProperties([Values(true, false)] bool canList,
-                                                     [Values(true, false)] bool canRead,
-                                                     [Values(true, false)] bool canWrite,
-                                                     [Values(true, false)] bool canDelete)
+        public async Task CanSetRepositoryProperties()
```

Comment:
Out of curiosity, why remove the different test cases?

Reply:
My thinking was that we're not testing the service, we're testing the client.  I can substantially shrink the size of my test recordings if I just validate once that the client can change all four values.  I can't think of any good reason to test all possible combinations.